response_operations_ui/views/collection_exercise.py

```diff
@@ -175,6 +175,12 @@ def view_collection_exercise(short_name, period):
     has_edit_permission = user_has_permission("surveys.edit")
     context = build_ce_context(ce_details, has_edit_permission, locked)
 
+    breadcrumbs = [
+        {"text": "Surveys", "url": url_for("surveys_bp.view_surveys")},
+        {"text": "Collection exercises", "url": url_for("surveys_bp.view_survey", short_name=short_name)},
+        {},
```

Comment:
I will raise a card for these empty dict down to how the breadcrumbs is currently implemented